extensions/src/test/java/org/pentaho/platform/web/http/api/resources/services/FileServiceTest.java

```diff
@@ -500,6 +501,32 @@ public void testGetDefaultLocationWhenHomeDirectoryIsHiddenAndNoDefaultFolderPro
     assertEquals( ClientRepositoryPaths.getPublicFolderPath(), fileService.doGetDefaultLocation( ":home:suzy" ) );
   }
 
+  @Test
+  public void testDoCanEdit() throws Exception {
```

Comment:
It would really help readibility if the three tests were separated by a new line.

Reply:
will update